ArduCopter/defines.h

```diff
@@ -80,14 +80,20 @@ enum tuning_func {
 #define WP_YAW_BEHAVIOR_LOOK_AT_NEXT_WP_EXCEPT_RTL    2   // auto pilot will face next waypoint except when doing RTL at which time it will stay in it's last
 #define WP_YAW_BEHAVIOR_LOOK_AHEAD                    3   // auto pilot will look ahead during missions and rtl (primarily meant for traditional helicopters)
 
-
 // Airmode
 enum class AirMode {
     AIRMODE_NONE,
     AIRMODE_DISABLED,
     AIRMODE_ENABLED,
 };
 
+// Force Flying
+enum class ForceFlying {
+    FORCEFLYING_NONE,
+    FORCEFLYING_DISABLED,
```

Comment:
Do we need to make the distinction between Disabled and none? We could make it a bool?